README.rst

```diff
@@ -202,6 +202,18 @@ Screenshots
    :target: https://cloud.githubusercontent.com/assets/122800/11951695/f371b926-a88a-11e5-91c2-499166776bd3.png
    :alt: Enhanced HTML report
 
+Contributing
+------------
+
+Fork the repository and submit PRs with bug fixes and enhancements,  contributions are very welcome.
+
+Tests can be run locally with `tox`_, for example to execute tests for Python 2.7 and 3.6 execute::
+
+    tox -e py27,py36
```

Comment:
It does work, I was just expecting tox to complain that it's not a valid environment. I'm fine with users running one or both of these environments before submitting a pull, so let's leave this as it is. Thanks for the patch!